### PR TITLE
Not 7

### PR DESCRIPTION
**Pull request by valentinaconde**

### CODE REVIEW

File: src/components/List/index.tsx [hunk 1]
```diff
@@ -4,9 +4,11 @@ import CloseIcon from '@mui/icons-material/Close';
 import './list.css'
 import EmptyData from '../EmptyData';
 import{ ConfirmAlert,  WarningAlert } from '../../services/SwalService';
+import { Note } from '../../types/Note';
+import { v4 as uuidv4 } from 'uuid';
 
 function List() {
-  const [list, setList] = useState<string[]>([])
+  const [list, setList] = useState<Note[]>([])
```
Comment (jonathanduran7) [inline src/components/List/index.tsx]: rename state to notes

File: src/components/List/index.tsx [hunk 2]
```diff
@@ -19,8 +21,14 @@ function List() {
   const handleSubmit = (event: any) => {
```
Comment (jonathanduran7) [inline src/components/List/index.tsx]: Find types correct for event

File: src/components/List/index.tsx [hunk 3]
```diff
@@ -66,7 +74,7 @@ function List() {
       {
         list.map((item, index) => (
```
Comment (jonathanduran7) [inline src/components/List/index.tsx]: When you have id should use it for index

**Review by jonathanduran7 — CHANGES_REQUESTED** [review]

File: src/components/List/index.tsx (hunk 1 above)
Comment (valentinaconde) [reply src/components/List/index.tsx]: ahh si lo pense

File: src/components/List/index.tsx (hunk 3 above)
Comment (valentinaconde) [reply src/components/List/index.tsx:75]: en la key burro no en el index

**Review by jonathanduran7 — APPROVED** [review]
excelente trabajo valentina!

**Pull request merged**